### ISSUE TITLE
travis error

### ISSUE THREAD

**Issue by loicjaouen** [dhlab-basel/Knora#504]
dear all, here is the strange error I received from a travis build that failed:

```
/home/travis/.travis/job_stages: line 53: /usr/local/bin/sbt: Permission denied
```

I am not familiar with travis yet, but I can add two piece of information that could be completely unrelated:
- earlier in the log, it says that it updates `sbt`
- travis has changed its base container image yesterday

**Comment by subotic** [comment]
Thanks. I will take a look.

**Comment by loicjaouen** [comment]
that's the build report:  
https://travis-ci.org/dhlab-basel/Knora?utm_source=email&utm_medium=notification

**Comment by subotic** [comment]
Fixed in PR #505
